### ISSUE TITLE
Days Off link takes you to wrong URL

### ISSUE THREAD

**Issue by CoolDadTx** [Microsoft/vsts-team-calendar#47]
When clicking the Days Off link on the right side of Team Calendar (0.3.11) it sends me to the URL (<collectionurl>/<team>/_backlogs/capacity/<sprint>. While that looks like the correct URL and shows the correct information, if you change the Days Off and then click the Save button you get a 401 error.  The actual URL that is being used by TFS when going to Capacity is <collectionurl>/<team>/_backlogs/capacity.

**Comment by hajohns** [comment]
Do both your team name and project name appear in the url, or is one or both left out?

**Comment by CoolDadTx** [comment]
It contains both the team project (FSMB) and the team name (FSMB Team).
Michael Taylor
http://www.michaeltaylorp3.net/
 

    On Monday, May 2, 2016 11:56 AM, hajohns <notifications@github.com> wrote:
 

 Do both your team name and project name appear in the url, or is one or both left out?—

**Comment by hajohns** [comment]
While the double // is not intended behavior, it should not affect the request. If you're getting a 401 trying to set Capacity, then my guess is you don't have access to set Capacity on that project for some reason.

**Comment by CoolDadTx** [comment]
I don't see how this could be a permissions issue.  I am the TFS admin and I set/copy the capacity for the entire team every sprint.
Nevertheless I applied the 0.3.17 update and days off appears to be working now.  Michael Taylor
http://www.michaeltaylorp3.net/
 

    On Wednesday, May 11, 2016 9:41 AM, hajohns <notifications@github.com> wrote:
 

 While the double // is not intended behavior, it should not affect the request. If you're getting a 401 trying to set Capacity, then my guess is you don't have access to set Capacity on that project for some reason. —
